baseband/vlbi_base/base.py

```diff
@@ -70,6 +71,31 @@ def __init__(self, fh_raw, header0, nchan, bps, complex_data, thread_ids,
 
         self.frames_per_second = frames_per_second
         self.offset = 0
+        self.squeeze = squeeze
+
+    @property
+    def sample_shape(self):
+        """Shape of a data sample (possibly squeezed).
+        """
```

Comment:
Move closing `"""` to previous line for single-line docstrings.

Reply:
Changed.